### ISSUE TITLE
Install Instructions

### ISSUE THREAD

**Issue by jlpoolen** [sirjaren/gentoo-overlay#6]
I recommend that since Gentoo is still at app-portage/layman 2.0.0-r1 that you provide this link in your README:
 
    "Using 'repos.conf' method (default method for app-portage/layman-2.3.0 or later )" => https://wiki.gentoo.org/wiki/Layman#Using_.27repos.conf.27_method_.28default_method_for_app-portage.2Flayman-2.3.0_or_later_.29

I have no problem going ahead of the currently approved version of layman and installing 2.3.0, but I would want to do so with some guidance and understanding, especially since in Layman, the "old" method of using make.conf is being supplanted by the new mthod of "repos.conf".

**Comment by jlpoolen** [comment]
Even after upgrading to layman 2.3.0, the conf_type is not in the file.  

    themis package.accept_keywords # eix -I layman
    [I] app-portage/layman
         Available versions:  2.0.0-r1 2.0.0-r3 ~2.1.0-r3 ~2.2.0-r7 (~)2.3.0 **9999 {bazaar cvs darcs g-sorcery +git gpg mercurial squashfs subversion sync-plugin-portage test PYTHON_TARGETS="pypy python2_7 python3_3 python3_4"}
         Installed versions:  2.3.0(04:07:42 AM 10/18/2015)(git -bazaar -cvs -darcs -g-sorcery -gpg -mercurial -squashfs -subversion -sync-plugin-portage -test PYTHON_TARGETS="python2_7 python3_4 -pypy -python3_3")
         Homepage:            http://layman.sourceforge.net
         Description:         Tool to manage Gentoo overlays

    themis package.accept_keywords # grep '^conf_type' /etc/layman/layman.cfg
    themis package.accept_keywords #

I'm guessing when layman is updated, it keeps the old method, so manual intervention is needed to adopt the new method.

**Comment by sirjaren** [comment]
Huh, very interesting.

I'm not an `layman` expert, but yes, I would agree with what your saying.

I guess at this point, I would recommend following the official guides and do what works for you on your version of `layman`.

[Layman](https://wiki.gentoo.org/wiki/Layman)
[Overlay](https://wiki.gentoo.org/wiki/Overlay)

I'm going to close this issue for now, but feel free to comment and/or re-open if need be.

Thanks jlpoolen.
